cypress/e2e/shared/flows.test.ts

```diff
@@ -295,7 +295,7 @@ describe('Flows', () => {
     cy.getByTestID('nav-item-flows').click()
     cy.reload()
     cy.getByTestID('tree-nav').should('be.visible')
-    cy.getByTestID('resource-editable-name').should('be.visible')
+    cy.getByTestID('resource-editable-name').should('exist')
```

Comment:
why was this changed required?

Reply:
because in the slimmed down cypress window its not visible due to the box of the presets, so we need it to _be_ there,but being visible wasn't possible

Reply:
how does the next line work then? Cypress' `.click()` only works if the element is visible

Reply:
it works even if its off screen, as far as i could see and local testing bore out that its ok

Reply:
huh ok! cypress must have changed that behavior then